### PR TITLE
Correct answers for blindness quiz section 1.5

### PR DESCRIPTION
**Pull request by virtual**

### CODE REVIEW

**Comment by james-priest** [comment]
omg. Thank you for the correction! Blindness is definitely a visual impairment rather than a motor 
one.😂😂

File: docs/course-notes/web-accessibility.md [hunk 1]
```diff
@@ -173,15 +173,15 @@ Which temporalities does the impairment **broken arm** fit into? (check all that
 ### 10.5 Quiz: Blindness
 #### Question 1 of 2
 Which categories does the impairment **blindness** fit into? (check all that apply)
-- [ ] Visual
-- [x] Motor
```
Comment (MuraliKrishnaGarimella) [inline docs/course-notes/web-accessibility.md:177]: The motor will have impairment broken arm

File: docs/course-notes/web-accessibility.md [hunk 2]
```diff
@@ -173,15 +173,15 @@ Which temporalities does the impairment **broken arm** fit into? (check all that
...
-- [ ] Visual
-- [x] Motor
+- [x] Visual
+- [ ] Motor
 - [ ] Hearing
 - [ ] Cognitive
 
 #### Question 2 of 2
 Which temporalities does the impairment **blindness** fit into? (check all that apply)
-- [x] Temporary
```
Comment (MuraliKrishnaGarimella) [inline docs/course-notes/web-accessibility.md:183]: Temporary

**Review by MuraliKrishnaGarimella — COMMENTED** [review]
done

**Pull request merged**